.github/workflows/CI.yml

```
@@ -74,7 +81,14 @@ jobs:
             })
 
 
-      - name: Build and analyze
+      - name: Docker build and push
+        run: |
+          echo "${{ secrets.DOCKER_HUB_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_HUB_USERNAME }}" --password-stdin
```

```suggestion
- name: Login to Docker Hub
      run: docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_ACCESS_TOKEN }}
```

로그인을 따로 빼면 간결해질 것 같아요

---

따로 분리할 생각을 못했었네요 감사합니다 반영하겠습니다